Add guidance for making the switch to PostHog (#11195)
PostHog/posthog.com: contents/docs/new-to-posthog/switch-guide/migration-planning.mdx

---
title: What to expect as you migrate
---

import { CalloutBox } from 'components/Docs/CalloutBox'

With your team onboard and a successful pilot completed, you can turn your attention to **planning your migration**.

## Technical planning

PostHog provides tools and guides to support you making this switch. Check out our migration section for details on moving from Amplitude, Mixpanel, LaunchDarkly and others.

Stumped on any detail of the process? If you’ve already signed up and started sending events with a forecasted bill of $500 or more, our **onboarding** team will reach out to help. If you haven’t heard from us yet, [drop us a line](/talk-to-a-human) with your migration questions!

<CalloutBox title="Avoid common setup pitfalls" type="caution">

Don’t create separate projects for mobile and web. PostHog works best when you can see user behavior across all versions of your products in one set of data and views.

Healthy integrations have a few things in common. It can help to review the [health check](/handbook/growth/sales/health-checks) documentation used by our sales team to tackle the most common ways to keep your bill manageable and predictable.

> If your historical data is above 50 GB, you should [reach out to our sales team](/talk-to-a-human) or [open a support ticket](/docs/support-options) so we can make sure everything goes through smoothly.

Author: > 50 GB

I chose this arbitrarily; lmk the scale we should indicate here

***

## Technical planning

PostHog provides tools and guides to support you making this switch. Check out our [migration docs](/docs/migrate) for details on moving from [Amplitude](/docs/migrate/migrate-from-amplitude), [Mixpanel](/docs/migrate/mixpanel), [LaunchDarkly](/docs/migrate/launchdarkly) and others.

Stumped on any detail of the process? If you’ve already signed up and started sending events with a forecasted bill of $500 or more, our **onboarding** team will reach out to help. If you haven’t heard from us yet, [drop us a line](/talk-to-a-human) with your migration questions!

<CalloutBox icon="IconWarning" title="Avoid common setup pitfalls" type="caution">

Don’t create separate projects for mobile and web. PostHog works best when you can see user behavior across all versions of your products in one set of data and views.

Healthy integrations have a few things in common. It can help to review the [health check](/handbook/growth/sales/health-checks) documentation used by our sales team to tackle the most common ways to keep your bill manageable and predictable.

> If your historical data is above 50 GB, you should [reach out to our sales team](/talk-to-a-human) or [open a support ticket](/docs/support-options) so we can make sure everything goes through smoothly.

Author: this ended up doing more harm than good, much as I like the premise

<img width="688" alt="Screenshot 2025-04-09 at 5 06 59 PM" src="https://github.com/user-attachments/assets/c36fd3ba-c320-41d1-b22c-d6f8ccfecb27" />

</CalloutBox>

### Reducing disruption for your teammates

New software can take a moment to click. One way to help your teammates get comfortable is making sure vital numbers are already waiting in your new PostHog setup.

> To support this, it can help to make a spreadsheet for people to request their most essential metrics. You want to know what data they need to keep making good decisions, and ensure those metrics are easy to find in PostHog. Capturing these requirements can also help you check that your technical implementation is handling everything necessary.

Reviewer: ```suggestion
To support this, make a spreadsheet for people to request their most essential metrics. Use it to create insights in PostHog and then add link them in the spreadsheet. You want to know what data they need to keep making good decisions, and ensure those metrics are easy to find in PostHog. 
```

Author: NICE

While moving historical events over is straightforward, you’ll need to take some time rebuilding the views and dashboards your team relies on.

### Conversion tracking

> An urgent need from marketing and product teams is **tracking conversions**. Getting clear on how to do this in PostHog can reduce frustration as everyone settles in. These docs on using **[funnels](/docs/product-analytics/funnels)** are a great starting point for this kind of reporting.

Reviewer: I think there is three parts to conversion tracking:

1. Web analytics conversion
2. Funnels
3. Sending conversion data to tools using pipelines

Do we want to cover all three here?

Author: I think for this stage I just want to make sure that the conversion tracking isn't *lost*, and make that feel scoped and achievable

"Oh, you lost my conversion data" is a great way to freak out whoever gets choked if the conversions go down

***

## Providing demos

> A great way to help PostHog land well is to offer a quick demo to everyone who’s interested. Remember, we don’t charge by the seat, so anyone who’s curious about user behavior in your product can log in at no extra charge.

Reviewer: This second point is repetitive and doesn't really add anything. 
```suggestion
A great way to help PostHog land well is to offer a quick demo to everyone who’s interested.
```

Here are some topics that might be worth covering. You know your teammates best, so be sure to adjust your demos to their needs and interests.

> ### Events, persons, properties and the activity view

Reviewer: ```suggestion
### Events, persons, properties, and the activity view
```

Author: rueful that I could pick up the endash house style immediately, but my internal compiler doesn't even catch the oxfords, thank you

![The activity explorer](https://res.cloudinary.com/dmukukwp6/image/upload/activity_2923c81b32.png)

Events are the core of PostHog’s reporting. Explain how they work, and the properties you plan to attach to them as part of your integration. Show off the **[activity](https://app.posthog.com/project/activity/explore)** view so everyone can see new events pouring in. 

If you’re identifying users in your integration, showing how **[person profiles](https://app.posthog.com/project/persons)** collect information can also be helpful.

### Web analytics

Everyone who wants the basics of an analytics dashboard will find it in our **[web analytics](https://app.posthog.com/project/web)** tab right out of the box.

### Actions

Actions let you group and rename events, making them easier to use in reporting. The more your teammates understand actions, the easier everyone’s life gets to be, so take a moment in **[data management](https://app.posthog.com/project/data-management/actions)** to explain.

### Creating insights and dashboards

> Creating, editing and duplicating [insights](/docs/product-analytics/insights) is fast and easy, and so is adding these to dashboards. Make sure to give this part of the demo plenty of detail: playing with data is a big part of what makes PostHog fun. This is also a great time to explore **filters**, which are a common need.

Reviewer: ```suggestion
Creating, editing, and duplicating [insights](/docs/product-analytics/insights) is fast and easy, and so is adding these to dashboards. Make sure to give this part of the demo plenty of detail: playing with data is a big part of what makes PostHog fun. This is also a great time to explore **filters**, which are a common need.
```

Popular insights include:

- Trends: show change over time  
- Funnels: track conversions and other sequential activity  
- User paths: see how real people explore your sight

### Replays

[Replays](https://app.posthog.com/project/replay/home) are a hit across roles and teams. For engineers, they can be a way of catching layout problems and other bugs early. For product and design, they’re a way to make sure that new features and workflows are actually performing correctly. They’re a huge help to support, and even execs enjoy flipping through them.

### Instant, no-code surveys

If you’re talking to someone who really wants to hear from customers, [surveys](https://app.posthog.com/project/surveys) are going to be exciting. Walk through how to design and deploy one.

### Toolbar

> This one is a visual crowd pleaser: launch your site’s PostHog [toolbar](https://app.posthog.com/project/toolbar) to show heatmap overlays and other stats directly on your site.

Reviewer: Why we don't have a docs page for this, I do not understand.
```suggestion
This one is a visual crowd pleaser: launch your site’s PostHog [toolbar](https://app.posthog.com/project/toolbar) to show heatmap overlays and other stats directly on your site. You can even use it to launch [hedgehog mode](/blog/aruba-hackathon#hedgehog-mode--toolbar--awesomeness).
```

</CalloutBox>

### Reducing disruption for your teammates

New software can take a moment to click. One way to help your teammates get comfortable is making sure vital numbers are already waiting in your new PostHog setup.

To support this, it can help to make a spreadsheet for people to request their most essential metrics. Use it to create insights in PostHog, then link them in the spreadsheet.

You want to know what data your teammates need to keep making good decisions, and ensure those metrics are easy to find in PostHog. Capturing these requirements can also help you check that your technical implementation is capturing all necessary events and properties.

While moving historical events over is straightforward, you’ll need plan for some time rebuilding the views and dashboards your team relies on.

### Conversion tracking

An urgent need from marketing and product teams is **tracking conversions**. Getting clear on how to do this in PostHog can reduce frustration as everyone settles in. These docs on using **[funnels](/docs/product-analytics/funnels)** are a great starting point for this kind of reporting.

***

> ## Providing demos

Reviewer 2: Not important for shipping V1, but I think "How to demo to your teammates" might better as it's own page. More than half this is page is about this, rather than the nuts and bolts of migrating. It get they're related, but as we build this out I think we can split them up.

Popular insights include:

- Trends: show change over time  
- Funnels: track conversions and other sequential activity  
- User paths: see how real people explore your sight

### Replays

[Replays](https://app.posthog.com/project/replay/home) are a hit across roles and teams. For engineers, they can be a way of catching layout problems and other bugs early. For product and design, they’re a way to make sure that new features and workflows are actually performing correctly. They’re a huge help to support, and even execs enjoy flipping through them.

### Instant, no-code surveys

If you’re talking to someone who really wants to hear from customers, [surveys](https://app.posthog.com/project/surveys) are going to be exciting. Walk through how to design and deploy one.

### Toolbar

This one is a visual crowd pleaser: launch your site’s PostHog [toolbar](https://app.posthog.com/project/toolbar) to show heatmap overlays and other stats directly on your site. You can even use it to launch [hedgehog mode](/blog/aruba-hackathon#hedgehog-mode--toolbar--awesomeness).

***

## Get more help

> Again, we’ve got your back here. We never want to waste your time with a call, so a self-serve migration is an open invitation. But if any part of this feels more daunting than fun, just [get in touch](/talk-to-a-human). We’re able to help accounts with $500 in projected monthly spending with onboarding, along with ongoing support for companies 25 people and up.

Reviewer 2: change this to a callout box at the end? I think it would look neater.
